matchms/filtering/add_retention.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any
+from typing import List
+from matchms.utils import get_first_common_element
+from ..typing import SpectrumType
+
+
+_retention_time_keys = ["retention_time", "retentiontime", "rt", "scan_start_time"]
+_retention_index_keys = ["retention_index", "retentionindex", "ri"]

```

Comment:
The retention time is sometimes given in minutes and sometimes in seconds (really annoying again...). It would be best if this would be handled by this function, but I don't have any good suggestions myself. 
So mostly a note that you are aware of this and maybe someone has a good suggestion how to solve this.

Reply:
I'm actually aware of this and dealt with that some times. I thought about making this eventually available as an optional parameter for the function.